docs/getting_started/install-on-mac-os-and-windows.md

```diff
@@ -0,0 +1,98 @@
+# Installation on macOS or Windows
+
+This page explains how to install [[= product_name =]] on macOS or Windows.
+
+!!! caution
+
+    This procedure is **for development purposes only**.
+    Installing [[= product_name =]] for production purposes is supported only on Linux.
+
+    If you want to use [[= product_name =]] in the production environment, see [Installing [[= product_name =]]](../getting_started/install_ez_platform.md).  
+
+### Prepare work environment
+
+To install [[= product_name =]], you need a stack with MySQL and PHP.
+Additionally, you need [Node.js](https://nodejs.org/en/) and [Yarn](https://yarnpkg.com/lang/en/docs/install/) for asset management.
+If you want to use a web server, you need to install it as well:
+
+- For Windows: Apache
+- For macOS: Apache/nginx
+
+The instructions below assume you are using Apache.
+
+??? "Windows"
+
+    Locate the `php.ini` file and open it in a text editor.
+    Provide missing values to relevant parameters, e.g. `date.timezone` and `memory_limit`:
+
+    ``` bash
+    date.timezone = "Europe/Warsaw"
+    memory_limit = 4G
+    ```
+
+    Uncomment or add extensions relevant to your project, e.g. `opcache` extension for PHP (recommended, not required):
+
+    ``` bash
+    zend_extension=opcache.so
+    ```
+
+    Edit Apache configuration file `httpd.conf`.
+    Replace placeholder values with corresponding values from your project, e.g. `ServerName localhost:80`.
+    Uncomment relevant modules, e.g.:
+
+    ``` bash
+    LoadModule rewrite_module modules/mod_rewrite.so
+    LoadModule vhost_alias_module libexec/apache2/mod_vhost_alias.so
+    ```
+
+    Start Apache by running: `httpd.exe`.
+
+    !!! note
```

Comment:
I would get rid of the note and move the "Install Apache as a Windows service..." step right before "Edit Apache configuration file" step. Then the step above could take the form of:

```
        Start Apache with:

        ``` bash
        httpd.exe -k start
        ```
  ```